src/schemas/config_schema_v1.json

```diff
@@ -73,7 +73,7 @@
           ]
         },
 
-        "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
+        "extra_hosts": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
```

Comment:
Please don’t do this, these schemas should be kept pristine, copied verbatim from upstream (docker-compose repo). If you need to ensure output has a specific type, either validate and reject in compose.ts/normalize() (like before) or silently convert. Output types are defined types.ts.

Reply:
Huh, I didn't know these where copied from there, I'll check the compose repo. I'll fix the normalization code instead then.

Reply:
Done @dfunckt I just fixed the normalization now (and added a test case and a fix for a random TS error that I'm not sure about)